packages/network/src/createNetwork.ts

```diff
@@ -10,7 +10,8 @@ import { Signer, Wallet } from "ethers";
 import { computedToStream } from "@latticexyz/utils";
 import { privateKeyToAccount } from "viem/accounts";
 import { Address, fallback, webSocket, http, createPublicClient, createWalletClient } from "viem";
-import * as chains from "viem/chains";
```

Comment:
why wagmi chains instead of viem chains?

Reply:
viem chains are pinned to a specific version (I thought it was a peer dep allowing any version), wagmi chains at least aligns with everything else using them